src/app/components/PostList.tsx

```diff
@@ -0,0 +1,62 @@
+'use client'
+import React, { useState, useEffect } from 'react';
+import { format } from 'date-fns';
+
+const PostList = ( { posts }: any ) => {
```

Comment:
You can create a type for posts, and set it for this function.
The type that is used for posts in Backend is the following: 
export interface IPost {
  id: string;
  title: string;
  description: string;
  likes: number;
  // comments              
  createdBy?: IUser;
  userId: string;
  createdAt: Date;
}

Comments is still in working, the updated field will be provided later on